tests/test_api.py

```diff
@@ -425,3 +425,46 @@ def test_remove_blobs_antialiased(self):
         mask = cv2.imread(image_path('map_noblobs_antialiased.png'))
         removed = histonets.remove_blobs(image, 0, 100, method='antialiased')
         assert np.array_equal(mask, removed)
+
+    def test_binarize_already_binary_image(self):
+        image = cv2.imread(image_path('map_ridges_invert.png'), 0)
+        binarized = histonets.binarize_image(image)
+        assert np.array_equal(image, binarized)
+
+    def test_binarize_already_binary_image_with_method(self):
+        image = cv2.imread(image_path('map_ridges_invert.png'), 0)
+        binarized = histonets.binarize_image(image, method='otsu')
+        assert np.array_equal(image, binarized)
+
+    def test_binarize(self):
+        image = cv2.imread(image_path('map.png'))
+        image_binarized = cv2.imread(image_path('map_bw.png'), 0)
+        binarized = histonets.binarize_image(image)
+        assert np.array_equal(image_binarized, binarized)
+
+    def test_binarize_default(self):
```

Comment:
Since the default is Li, renaming either this test or the previous may clarify the intent of the two tests a bit more.